packages/admin-ui/src/pages/packages/components/Network/HttpsMappings.tsx

```diff
@@ -107,6 +107,19 @@ export function HttpsMappings({
     }
   }
 
+  if (mappings.error && mappings.error.message.includes("ENOTFOUND"))
+    return (
+      <>
+        <p>
+          To do HTTPS networking is necessary to install the system package
+          HTTPS.
+        </p>
+        <br />
+        <a href="http://my.dappnode/#/installer/https-portal.dnp.dappnode.eth">
+          Get HTTPS package
+        </a>
```

Comment:
The ADMIN UI is React single page application and all navigation is done programatically. Look around to found examples with `react-router` components